verticapy/sql/geo/functions.py

```diff
@@ -61,6 +61,69 @@ def coordinate_converter(
     -------
```

Comment:
For everything below (all geo files):
 - Try to avoid import *. Import only the needed function or a module (then module.my_fun).
 - Try to create multiple block and bring the comment out when it is possible.
 - Be consistent: Use `display` when you want to display a result.
 - PEP8 for indentation
 - a small sentence to explain what you do.